provider/datagen/src/transform/cldr/fallback/mod.rs

```diff
@@ -62,6 +67,44 @@ impl DataProvider<LocaleFallbackParentsV1Marker> for crate::DatagenProvider {
     }
 }
 
+impl DataProvider<CollationFallbackSupplementV1Marker> for crate::DatagenProvider {
+    fn load(
+        &self,
+        _req: DataRequest,
+    ) -> Result<DataResponse<CollationFallbackSupplementV1Marker>, DataError> {
+        // TODO(#1964): Load this data from its proper sources. For now, it is copied from:
+        // https://github.com/unicode-org/icu/blob/main/tools/cldr/cldr-to-icu/build-icu-data.xml
+        // as well as from CLDR XML.
```

Comment:
This source also has:
`<forcedAlias source="sr_ME" target="sr_Cyrl_ME"/>`

Is that handled elsewhere?

Reply:
Will research this further and follow up in #1965. If I need to add more aliases, I can do this in patch releases without breaking data structure stability.